sktime/classification/deep_learning/tapnet.py

```diff
@@ -254,19 +254,20 @@ def get_test_params(cls, parameter_set="default"):
             `create_test_instance` uses the first (or only) dictionary in `params`.
         """
         param1 = {
-            "n_epochs": 50,
-            "batch_size": 32,
+            "n_epochs": 20,
+            "batch_size": 4,
             "use_lstm": False,
             "use_att": False,
-            "filter_sizes": (128, 128, 64),
+            "filter_sizes": (16, 16, 16),
             "dilation": 2,
-            "layers": (50, 25),
+            "layers": (32, 16),
         }
-
         param2 = {
-            "n_epochs": 100,
-            "use_cnn": False,
-            "layers": (25, 25),
+            "n_epochs": 20,
```

Comment:
Is there a reason for including convolutional layers here? Since we're already covering it in the first param set, we could directly set `use_cnn: False` since the current set is a subset (and hence more resource-hungry) than what I'm proposing.

The current set is good as well, I was just thinking in terms of improving coverage.

Reply:
changed to set use_cnn to False